src/migrations/migration_steps.ts

```diff
@@ -548,6 +548,32 @@ migrationStepRegistry
       }
       return data;
     },
+  })
+  .add("19.0", {
+    migrate(data: WorkbookData): any {
+      for (const sheet of data.sheets || []) {
+        const borderType: Record<UID, Zone[]> = {};
+        for (const zoneXc in sheet.borders) {
+          const borderId = sheet.borders[zoneXc];
+          if (!borderId) continue;
+          if (!(borderId in borderType)) borderType[borderId] = [];
+          borderType[borderId].push(toZone(zoneXc));
+        }
+        const borders = {};
+        for (const borderId in borderType) {
+          const border = data.borders[borderId];
+          if (!border) continue;
+          const zones = recomputeZones(borderType[borderId]);
+          if (border.left || border.right) border.vertical = border.left || border.right;
```

Comment:
Can we have `border.left = 0` or `border.top = 0` ? If so, we should compare to `undefined`.